web/models.py

```
@@ -2804,3 +2805,417 @@ class ScheduledPost(models.Model):
+            self.approved_by = approver
+            self.save()
+
+            # Create notification for the user
+            notification_title = "Chapter Membership Approved"
+            notification_message = f"Your membership to {self.chapter.name} has been approved as {self.get_role_display()}."
+
+            Notification.objects.create(
+                user=self.user,
+                title=notification_title,
+                message=notification_message,
+                notification_type="success"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)